src/com/xilinx/rapidwright/design/blocks/PBlockGenerator.java

```diff
@@ -940,16 +940,57 @@ public static void main_testing(String[] args) {
 					}
 					if(p.get(pIdx) == t) pIdx++;
 				}
-				sb.append("SLICE_X" + upperLeft.getInstanceX() + "Y" + (upperLeft.getInstanceY()-(numSLICERows-1)) + 
-						 ":SLICE_X" + (upperLeft.getInstanceX()+(numSLICEColumns+numSLICEMColumns)-1) + "Y" + upperLeft.getInstanceY());
+				
```

Comment:
This update is clearly attempting to address the issue of the pblock colliding with the PS which is not a homogeneous--this is a problem.  Since we have a number of situations similar to this, perhaps its best if we hold off on these point solutions and work together on a more holistic approach.